modules/promotions/admin-menu-items/custom-fonts-promotion-item.php

```diff
@@ -19,31 +19,26 @@ public function get_page_title() {
 		return esc_html__( 'Custom Fonts', 'elementor' );
 	}
 
-	public function get_image_url() {
-		return ELEMENTOR_ASSETS_URL . 'images/go-pro-wp-dashboard.svg';
-	}
-
-	public function get_cta_text() {
```

Comment:
It would be helpful if we can leave this CTA button override for hosting's use.

Reply:
@Barcov 
this is not related to this PR